src/BUILD.gn

```diff
@@ -92,6 +93,11 @@ if (chip_build_tests) {
       deps += [ "${chip_root}/src/ble/tests" ]
     }
 
+    if (!is_clang && chip_device_platform != "darwin" &&
```

Comment:
Why doesn't it work with gcc?

Reply:
Opened an issue related to this: https://github.com/project-chip/connectedhomeip/issues/9173